torchaudio/prototype/functional/functional.py

```diff
@@ -48,7 +57,7 @@ def fftconvolve(x: torch.Tensor, y: torch.Tensor, mode: str = "full") -> torch.T
     Args:
         x (torch.Tensor): First convolution operand, with shape `(..., N)`.
         y (torch.Tensor): Second convolution operand, with shape `(..., M)`
-            (leading dimensions must match those of ``x``).
+            (leading dimensions must be broadcast-able to those of ``x``).
```

Comment:
should we follow the broadcasting rules more closely, e.g. accept inputs of different lengths and check each dimension starting from second to last trailing dimension?

Reply:
or, perhaps we could additionally clarify that the lengths of the leading dimensions need to be equal

Reply:
https://pytorch.org/docs/stable/notes/broadcasting.html ?

Let me look into that. I did not know about the case of different length.

Reply:
> or, perhaps we could additionally clarify that the lengths of the leading dimensions need to be equal

I'd go with firstly land this and followup for full broadcast support.

Reply:
can you also make the same change to the docstring for `torchaudio.prototype.transforms.FFTConvolve`?

Reply:
done